qsimcirq_tests/qsimcirq_test.py

```diff
@@ -1485,12 +1485,22 @@ def test_cirq_qsim_global_shift():
     cirq_result = simulator.simulate(circuit)
 
     qsim_simulator = qsimcirq.QSimSimulator()
-    qsim_result = qsim_simulator.simulate(circuit)
+    qsim_result1 = qsim_simulator.simulate(circuit)
 
     assert cirq.linalg.allclose_up_to_global_phase(
-        qsim_result.state_vector(), cirq_result.state_vector()
+        qsim_result1.state_vector(), cirq_result.state_vector()
     )
 
+    qsim_simulator.qsim_options["z"] = True
```

Comment:
How can I do that?

Reply:
Similar to other tests in this file, for example: https://github.com/quantumlib/qsim/blob/df527863af059e082dab3c8ba39ae779d32c71ec/qsimcirq_tests/qsimcirq_test.py#L1012-L1013

But with `denormals_are_zeros` instead of `max_fused_gate_size`, of course.

Reply:
Thanks. Is it possible to do that without creating a new simulator object?

Reply:
Ah, I see the issue here. No, to change the options on an existing simulator what you're currently doing is correct. This seems fine to me.